openedx/features/edly/utils.py

```diff
@@ -261,3 +268,55 @@ def send_comments_reply_email_to_comment_owner(comment, context):
     if parent_comment.user_id != comment.thread.user_id and parent_comment.user_id != comment.user_id:
         recipients = [parent_comment.user_id]
         send_bulk_mail_to_students.delay(recipients, message_context, 'comment_reply')
+
+
+def get_course_name_from_id(course_id):
+    try:
+        course_key = CourseKey.from_string(course_id)
+        return CourseOverview.get_from_id(course_key).display_name
+    except Exception:
+        return None
+
+
+def send_contact_us_email(data):
+    recipient_list = settings.CONTACT_US_SUPPORT_EMAILS
+    current_site = get_current_site()
+    message_context = get_base_template_context(current_site)
+    course_name = get_course_name_from_id(data['custom_fields'][0]['value'])
+    if course_name:
+        message_context.update({'course_name': course_name})
+    message_context.update({
```

Comment:
nit: we generally add a new line before de-indenting to increase readability.

Reply:
done